### ISSUE TITLE
Consider adding spells

### ISSUE THREAD

**Issue by ethansaxenian** [JDBumgardner/stone_ground_hearth_battles#54]
With the introduction of blood gems in the latest patch, we might want to consider adding a new card type for spells.

These are all the spells currently in the game:

Triple Rewards (golden cards, AFKay)
Prizes - functionally the same as Triple Rewards (Silas Darkmoon)
Gold Coins (Lord Barov, Lich Baz'hial, Warden of Old)
Bananas (King Mukla)
Big Bananas (King Mukla)
Recruitment Map (Elise Starseeker)
Blood Gems (8 new cards/heros)

Not to mention all the spells (30 different spells!) generated from the hero Tickatus, who is not even implemented yet.

We would definitely need to make some big changes to the neural net, but this solution feels much cleaner than having each spell be its own separate action.

**Comment by jeremysalwen** [comment]
Yeah this definitely makes sense, and I think it would be better for the neural net too!
